app/profile/SkillsTab.tsx

```diff
@@ -21,8 +32,48 @@ export const SkillsTab = () => {
     setModel({ ...model!, skills: encodeSkillSelection(selection) });
   };
 
+  const enableSubmitButton = bioState.submissionState === "changes_done";
+
+  const submitButtonLabel =
+    bioState.submissionState === "submitted_successfully"
+      ? "Úspěšně uloženo 🎉"
+      : "Uložit změny";
+
   return (
     <section className="mb-10">
+      <p className="mb-4">Řekni něco málo o sobě, ať tě lidé lépe poznají</p>
+      <textarea
+        className="block w-full max-w-prose rounded-md border-2 border-gray p-2"
+        rows={3}
+        defaultValue={actualBio}
+        onChange={(e) =>
+          setBioState(() => ({
```

Comment:
Je tady důvod nastavovat nový stav tou funkcí? Nemůžeš tam rovnou dát ten stav?

```typescript
setBioState({
    submissionState: "changes_done",
    bio: e.target.value,
})
```

Reply:
jasne to mi len nejak uslo :) dovod nie je